charts/gardener/controlplane/charts/application/templates/rbac-user.yaml

```diff
@@ -144,7 +211,28 @@ rules:
   - settings.gardener.cloud
   - operations.gardener.cloud
   resources:
-  - '*'
+  - backupbuckets
+  - backupentrie
+  - cloudprofiles
+  - controllerdeployments
+  - controllerinstallations
+  - controllerregistrations
+  - exposureclasse
```

Comment:
typo... Can we rather keep using `*` here for the resources (there are other examples in this file you didn't touch)?!